### PR TITLE
Fix toolgun not being able to scrollwheel

### PR DESCRIPTION
**Pull request by homonovus**
Fixes Wire's STool and many other STools not being able to scroll through options

### CODE REVIEW

File: plugins/wepselect.lua [hunk 1]
```diff
@@ -124,27 +124,37 @@ if (CLIENT) then
...
-			bTool = tool and (tool.Scroll != nil)
+		local currentWeapon = client:GetActiveWeapon()
+		local bValid = currentWeapon:IsValid()
+
+		if (bValid and currentWeapon:GetClass() == "weapon_physgun" and client:KeyDown(IN_ATTACK)) then
+			return
 		end
 
 		local weapons = client:GetWeapons()
+		if #weapons == 0 then return end
```
Comment (garytate) [inline plugins/wepselect.lua:147]: format to consistent style with helix

https://cs.github.com/NebulousCloud/helix/blob/4d1922407c93828cfd951aec70f89e834ee0f836/gamemode/core/libs/cl_hud.lua#L7
https://cs.github.com/NebulousCloud/helix/blob/4d1922407c93828cfd951aec70f89e834ee0f836/gamemode/core/libs/sh_storage.lua#L314

File: plugins/wepselect.lua [hunk 2]
```diff
@@ -153,7 +163,11 @@ if (CLIENT) then
 			end
 
 			return true
-		elseif (bind:find("invnext") and !bTool) then
+		elseif (bScrollDown) then
+			if (bHoldingToolgun and tool and tool.Scroll and tool:Scroll(tr, 0)) then
```
Comment (garytate) [inline plugins/wepselect.lua:167]: Duplicate with L154, can pull out of the scroll direction conditional?

Comment (homonovus) [reply plugins/wepselect.lua:167]: if bHoldingToolgun block is only hit once, if bScrollDown or bScrollUp is true
if it's not inside the scroll conditionals, it'll constantly be scrolling

Comment (garytate) [reply plugins/wepselect.lua:169]: Yeah that makes sense, would be quite a long single conditional

**Comment by homonovus** [comment]
ready for final review

**Pull request closed without merging**